datafusion/core/src/physical_plan/aggregates/row_hash.rs

```diff
@@ -107,22 +107,6 @@ pub(crate) struct GroupedHashAggregateStream {
     indices: [Vec<Range<usize>>; 2],
 }
 
-#[derive(Debug)]
```

Comment:
I have moved functions common for both in `row_hash.rs` and `bounded_aggregate_stream.rs` to the inside `util.rs` file. All the changes in this file, `util.rs` comes because of this reason.